### ISSUE TITLE
Accessibility - Is there any way to to navigate using keys (keyboard)

### ISSUE THREAD

**Issue by convpal** [moroshko/react-autosuggest#829]
I want a user to be able to show more content on the autosuggest dropdown just like Amazon's search bar, and I am able to show more content on the search bar, but unable to give more ability in terms of accessibility like on amazon's search bar.

Amazon sample:
![Screen Recording 2021-10-11 at 8 24 26 AM](https://user-images.githubusercontent.com/84512635/136727398-b8c70764-5a74-448e-9456-51e67dc2dcd4.gif)

Application search bar:

![image](https://user-images.githubusercontent.com/84512635/136727223-81cf637b-5429-4f3d-9b30-f5b16e930110.png)

Is there a way to achieve this effect? Any workarounds you can think off?

**Comment by KyleWiteck** [comment]
This works on the CDN example in https://react-autosuggest.js.org/, but the library does not. This is a requirement of the web now. Please get this fixed. 

In the meantime, do you have any work arounds or pointers on how to implement this?

**Comment by willbutler** [comment]
The examples appear to be using an older version of `react-autosuggest`. The latest 9.x version (9.4.3) seems to work.
